bots/pr/src/test/java/org/openjdk/skara/bots/pr/CheckTests.java

```diff
@@ -1350,7 +1350,7 @@ void excessiveFailures(TestInfo testInfo) throws IOException {
     }
 
     @Test
-    void useJCheckConfFromTargetBranch(TestInfo testInfo) throws IOException {
+    void invalidUpdatedJCheckConf(TestInfo testInfo) throws IOException {
```

Comment:
Now, this test duplicates `testRunJcheckTwiceWithBadConfiguration`, but I don't think it would hurt to keep it.